src/network/network.cpp

```diff
@@ -2237,6 +2237,42 @@ void game_command_kick_player(int *eax, int *ebx, int *ecx, int *edx, int *esi,
 	*ebx = 0;
 }
 
+void game_command_adjust_client_prefs(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp)
+{
+	uint8 playerid = (uint8)*eax;
```

Comment:
It is always much nicer to the next person if you make the first part of your game command assignments of all of the register pointers. I.e. I want to see uint8 flags = *ebx, userprefs = *edx, ...